src/package/type.rs

```diff
@@ -26,6 +26,18 @@ pub enum PackageType {
     Api,
 }
 
+impl TryFrom<i32> for PackageType {
```

Comment:
If you want to use this for protobuf handling, we should definitely impl 

`From<crate::proto::buffrs_registry::package::PackageType> for PackageType`

You can then go from `i32 -> crate::proto::buffrs_registry::package::PackageType -> PackageType`

The reason im flagging this is because if the proto file changes, this impl will stay the same which might lead to bugs. If we impl type conversion on the generated code stubs we don't convert explicitly but let tonic generate this code based on the proto file